src/components/domain/project/hooks/useConfirmDeleteProject.tsx

```diff
@@ -0,0 +1,44 @@
+import { Trash2 } from 'lucide-react';
+
+import { useModalStore } from '@/stores/modalStore';
+import MessageBox from '@/components/common/messgeBox/MessageBox';
```

Comment:
여기 지금 보니까  폴더명에 오타가 있는 것 같네요! `messge` -> `message`
```suggestion
import MessageBox from '@/components/common/messageBox/MessageBox';
```

Reply:
71aab8b636953c8c72390ea2f367bacc3ff457b8